app/src/main/java/com/example/yard/Activities/SignupActivity.java

```diff
@@ -0,0 +1,105 @@
+package com.example.yard.Activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.yard.R;
+import com.parse.ParseInstallation;
+import com.parse.ParseUser;
+
+import java.util.Arrays;
+
+public class SignupActivity extends AppCompatActivity {
+  // HBCUs in Tennesse, Virginia, and Delaware
+  final String[] schools = {
```

Comment:
`private static`. You can leave it here now for this PR. We'll learning modifier fundamentals and make a follow-up PR.